### PR TITLE
Adds the drink Lean and a recipe for it into the game.

### PR DESCRIPTION
**Pull request by Tinigame**
#### Summary
Content "Adds the recreational drug drink "Lean" into the game."

#### Purpose of change
Added this drink because it gives a way to consume cough syrup in a more fun way and to add more variety to the drink recipes in the game

#### Describe the solution
it adds Lean into the game as a fluid and as a recipe.

#### Describe alternatives you've considered
perhaps i could have made lean more "fun" in a different way but not exactly sure how i could.

#### Testing
Made sure the item had the effects it should by having a player character drink it in the game and made sure the recipe for the item is correct ingame.

#### Additional context
None

### CODE REVIEW

**Comment by fairyarmadillo** [comment]
Lean isn't made out of over-the-counter Nyquil or what have you, it's made from promethezine with codeine, which is available by prescription only and in many locations in the US isn't even kept in pharmacies.

[Source here](https://archives.drugabuse.gov/emerging-trends/syrup-purple-drank-sizzurp-lean).

**Comment by ERRORCODE509** [comment]
Damn, this is 'bouta obsolete my mod.

**Comment by GuardianDll** [comment]
I don't think lean is so well known you can autolearn it.

**Comment by Tinigame** [comment]
Because there isn't a prescription cough syrup in the game could lean be added then? Or could i perhaps tweak the recipe to include codeine pills to make it more realistic? Before i made the recipe for the game i thought every cough syrup contained codeine so that's an error on my part. The poppy syrup was added to the recipe so you could make this eve when you cant find cough syrup but it doesn't contain any euphoric elements as you said so ill tweak that as well.
Basically what im asking is could this even be an addition to the game because stronger cough syrup isn't in the game?

**Comment by fairyarmadillo** [comment]
I think the appropriate thing to do would be to add a promethazine with codeine syrup to the game, giving it stats that reflected its narcotic effects, and adding that to hospital and pharmacy back room itemgroups, provided it's not especially restricted in New England like it is in some states. Then you could make a lean recipe which would increase its enjoyability. Poppy syrup should probably be left out of this, as it's not an enjoyable drug, adding soda would affect the taste, but it still doesn't get people high.

**Comment by Tinigame** [comment]
Here i was thinking i could easily add lean into the game but here i am. I guess i could try also adding the promethazine syrup into the game aswell and tweak the recipe accordingly.

**Comment by Maleclypse** [comment]
Promethazine is a potentiator for opiates which I think could be shown in our effect system. And while promethazine codeine cough syrup isn’t that common anymore I’ve never encountered any problems getting either codeine or promethazine syrup filled at pharmacies.

**Comment by kevingranade** [comment]
Pass, as others have pointed out, the realities that led to lean being a thing are no longer present, I'm not seeing a reason to add this other than it being a meme, and that's not a good enough reason.
